app/views/layouts/history.html.arb

```diff
@@ -0,0 +1,13 @@
+panel "History" do
+  table_for assigns[:versions] do
+    column ("Item") do |v| 
+        if v.item
+           link_to v.item.id, [:admin, v.item]
+        end
+    end
+    column ("Type") { |v| v.item_type.underscore.humanize }
+    column ("Modified at") { |v| v.created_at.to_s :long }
+    column ("Text") { |v| v.object.to_s }
```

Comment:
arb! wow.
If diff is needed (not sure, but sounds useful), `v.changeset` will do that (if you add a `object_changes` column to help it).